ddtrace/internal/telemetry/data.py

```diff
@@ -15,6 +15,9 @@
 from ..hostname import get_hostname
 
 
+_platform = platform.platform(aliased=True, terse=True)
```

Comment:
```suggestion
_platform = callonce(lambda: platform.platform(aliased=True, terse=True))
```
Can we use the `callonce()` utility to ensure we only ever call this once and cache the result?

Reply:
We'd need to import `from ddtrace.internal.utils.cache import callonce` as well.